erpnext/selling/doctype/customer/customer.js

```diff
@@ -140,9 +140,53 @@ frappe.ui.form.on("Customer", {
 		var grid = cur_frm.get_field("sales_team").grid;
 		grid.set_column_disp("allocated_amount", false);
 		grid.set_column_disp("incentives", false);
+
+		if(frm.doc.__onload && frm.doc.__onload.dashboard_info) {
+			var company_wise_info = frm.doc.__onload.dashboard_info;
+			frm.set_value("total_monthly_sales", company_wise_info[0].billing_this_month);
+
+			if(frm.doc.total_monthly_sales && frm.doc.monthly_sales_target){
```

Comment:
I think this should just be checking for a set target, since the progress bar should show up even if there are no sales for the month.